packages/app/src/debug/empty/DebugEmptyView.vue

```diff
@@ -2,17 +2,23 @@
   <div class="flex flex-col mx-auto my-45px max-w-680px items-center">
     <div class="flex flex-col items-center justify-evenly">
       <div><i-cy-box-open_x48 class="icon-dark-gray-500 icon-light-indigo-100" /></div>
-      <div class="flex flex-col mx-[20%] mt-25px mb-20px items-center">
+      <div class="flex flex-col mt-25px mb-20px max-w-640px items-center">
         <div class="font-medium my-5px text-center text-gray-900 text-18px">
           {{ title }}
         </div>
         <div class="font-normal my-5px text-center leading-relaxed text-16px text-gray-600">
-          {{ description }} <ExternalLink
-            v-if="helpLinkText && helpLinkHref"
-            :href="helpLink"
-          >
-            {{ helpLinkText }}
-          </ExternalLink>
+          {{ description }}
+          <span class="ml-4px">
+            <ExternalLink
+              v-if="helpLinkHref"
```

Comment:
Should the `v-if` go on the `span` so we don't render the spacing unless it's needed?